gcn/notices/svom/alert.schema.json

```
@@ -0,0 +1,36 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/svom/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "unevaluatedProperties": false,
+  "title": "SVOM Alert",
+  "description": "ECLAIRs or GRM trigger parameters computed onboard",
+  "allOf": [
+    { "$ref": "../core/AdditionalInfo.schema.json" },
+    { "$ref": "../core/Alert.schema.json" },
+    { "$ref": "../core/DateTime.schema.json" },
+    { "$ref": "../core/Event.schema.json" },
+    { "$ref": "../core/Localization.schema.json" },
+    { "$ref": "../core/Reporter.schema.json" },
+    { "$ref": "../core/Statistics.schema.json" }
+  ],
+  "properties": {
+    "$schema": true,
+    "notice_level": {
+      "enum": ["N1e", "N1g"],
+      "description": "SVOM notice reference id. N1e is an ECLAIRs trigger, N1g a GRM trigger."
+    },
+    "slew_status": {
+      "enum": ["accepted", "rejected", "not_requested"],
+      "description": "Spacecraft slew status"
+    },
+    "trigger_type": {
+      "enum": ["count_rate", "image"],
+      "description": "In case of an ECLAIRs trigger, specifices whether count rate or image trigger"
+    },
+    "triggered_GRDs": {
+      "enum": ["111", "110", "101", "011", "100", "010", "001"],
+      "description": "In case of a GRM trigger, trigger status the 3 GRDs. Each digit correspond to one of the 3 GRDs. 1 means the GRD triggers, 0 not"
+    }
```

Rather than an enum, how about an array of booleans?

```suggestion
    "triggered_GRDs": {
      "type": "array",
      "items": {
        "type": "boolean"
      },
      "minItems": 3,
      "maxItems": 3,
      "description": "In case of a GRM trigger, trigger status the 3 GRDs. Each item correspond to one of the 3 GRDs. true means the GRD triggers, false not"
    }
```